training/src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,93 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+
+@Entity
+public class Users {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
Be careful with the cascade type. The possibilities are _persist, delete, merge_ and _referesh_.
**Persist** says: If you persist the Users instance to the DB, then persist every item that this collection has in the DB as well.
**Delete** says: If you delete the Users instance from the DB, then delete every item that this collection has in the DB as well.
**Merge** says: If you changed the in-memory values of the collection, then merge this changes to the DB so these are reflected there.
**Refresh** says: If something changed in the database values of the collection, then refresh those changes in the in-memory values.
We need to use **merge** and **refresh**